### ISSUE TITLE
Graphs

### ISSUE THREAD

**Issue by Arya766** [hw-manager/android#17]
It would be very nice if users could measure their productivity, in a selected preiod.

**Comment by AltNico** [comment]
Yeah, that would be nice, but it hasn't a high priority at the moment.

**Comment by Arya766** [comment]
Allow me not to think of priorities; it will neutralize my creativity. I tell you all, you decide what, when, where, etc. I just scout and present.
